src/services/chris_integration.tsx

```diff
@@ -131,8 +131,16 @@ class ChrisIntegration {
     return true;
   }
 
-  static async processOneImg(img: DcmImage, chosenXrayModel: string, chosenCTModel: string): Promise<BackendPollResult> {
-    let client: any = await ChrisAPIClient.getClient();
+  /**
+   * Runs pl-med2-img and pl-covidnet/pl-ct-covidnet on a DcmImage, given its pl-dircopy instance obtained from runDircopy
+   * @param img DcmImage
+   * @param dircopyPluginInstance instance obtained from runDircopy
+   * @param chosenXrayModel name of model to be used for pl-covidnet
+   * @param chosenCTModel name of model to be used for pl-ct-covidnet
```

Comment:
Good work on adding the documentation to the function. However, make sure it follows the JSDoc standard.

https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns

```suggestion
   * @param {DcmImage} img - <add description>
   * @param {string} chosenXrayModel - name of model to be used for pl-covidnet
   * @param {string} chosenCTModel - name of model to be used for pl-ct-covidnet
```

Reply:
Also, seems like "@param dircopyPluginInstance instance obtained from runDircopy" is outdated and was added before I made a comment about not needing to separate pl-dircopy from the rest of the plugins.